jaxrs/src/test/scala/com/cerner/beadledom/jaxrs/JaxRsParamConditionsSpec.scala

```diff
@@ -17,8 +17,8 @@ class JaxRsParamConditionsSpec extends FunSpec with ShouldMatchers {
           JaxRsParamConditions.checkParam(false, "some message")
         }
         exception.getResponse.getStatus should be(400)
-        exception.getResponse.getEntity should be("some message")
         exception.getResponse.getMediaType should be(MediaType.TEXT_PLAIN_TYPE)
+        exception.getMessage should be("some message")
```

Comment:
updated the matcher: 201987b

I think that the description in the it block is still valid.